vars/sectionDeployToAKS.groovy

```diff
@@ -57,7 +57,7 @@ def call(params) {
 
   GithubAPI gitHubAPI = new GithubAPI(this)
   def testLabels = gitHubAPI.getLabelsbyPattern(env.BRANCH_NAME, 'enable_')
-  def enableHelmLabel = gitHubAPI.checkForLabel(env.BRANCH_NAME, 'enable-helm')
+  def enableHelmLabel = gitHubAPI.checkForLabel(env.BRANCH_NAME, 'enable_helm')
```

Comment:
you already have the label from `testLabels` you just need to see if it's in that list and set a boolean value